src/query-builder/InsertQueryBuilder.ts

```diff
@@ -249,7 +249,7 @@ export class InsertQueryBuilder<Entity> extends QueryBuilder<Entity> {
       if (statement && statement.overwrite instanceof Array) {
         if (this.connection.driver instanceof MysqlDriver) {
           this.expressionMap.onUpdate.overwrite = statement.overwrite.map(column => `${column} = VALUES(${column})`).join(", ");
-        } else if (this.connection.driver instanceof PostgresDriver) {
+        } else if (this.connection.driver instanceof PostgresDriver || this.connection.driver instanceof AbstractSqliteDriver) {
           this.expressionMap.onUpdate.overwrite = statement.overwrite.map(column => `${column} = EXCLUDED.${column}`).join(", ");
```

Comment:
hm so confusing, is it onUpdate or orUpdate? Looks like second, not sure who named it onUpdate before then.

Reply:
I could take advantage of this PR and change the key within `expressionMap` to `orUpdate` if you see it ok.

Reply:
we can left it as it is right now, and change it later (do it if you want), I just commented on what confused me...